src/wasi/WASI.cpp

```diff
@@ -118,17 +118,17 @@ void WASI::args_get(ExecutionState& state, Value* argv, Value* result, Instance*
 
     TemporaryData<void*, 8> pointers(argc);
 
-    if (pointers.data() == nullptr) {
```

Comment:
These checks are unnecessary, because uvwasi checks its input pointers. Instead, we check that the result is success.